faqs/emergency-shutdown.md

```diff
@@ -0,0 +1,130 @@
+
+## During Emergency Shutdown, will CDP owner's collateral be given away to other people?
+
+During Emergency Shutdown, collateral is immediately made available for redemption by Dai holders. This means that someone may receive a part of your original collateral. However, after the time period where CDP collateral claims are processed, any CDP owner will be able to withdraw collateral that represents the full net value of their CDP.
+
+Example:
+
+If you have a CDP with 1 ETH that's worth 300 USD, and 100 debt, you will be left with 0.6666 ETH and no debt. The Net Value of the CDP is made available for redemption, which is 300 USD value of the collateral, minus 100 USD of debt. Therefore, you will be able to claim 0.6666 ETH after the 6 hour delay period.
+
+## Is there anything preventing MKR holders from triggering an Emergency Shutdown in order to avoid MKR dilution due to poor management?
+
+MKR holders, especially large ones, are incentivized to preserve the value of their MKR. Voting to perform an Emergency Shutdown on the system because of the fear of imminent dilution may harm the value of their MKR. It also puts their MKR ownership at risk, as anyone can redeploy the system and change the token distribution of MKR. Therefore, this is a very risky attack vector.
+
+## Can I create a CDP during an Emergency Shutdown?
+
+No.
+
+## What happens to inaccessible CDPs and Dai that has been lost, stuck or is dust?
+
+Any Dai or CDPs that have been stuck or lost will remain so afterward, they will effectively be stuck or lost ETH instead.
+
+## Are Stability Fees waived in the event of an Emergency Shutdown?
+
+Stability Fees are not paid in an Emergency Shutdown, but governance can enact an "Emergency Shutdown Penalty" that will ensure that anyone who holds their CDP through Emergency Shutdown rather than migrating it to MCD will have to pay a higher penalty, meaning that it is always cheaper to migrate your CDP and pay the stability fee.
+
+## Has an Emergency Shutdown ever been performed in the past?
+
+Yes, an Emergency Shutdown has been [performed](https://www.reddit.com/r/MakerDAO/comments/7lhbmx/psa_sai_now_globally_settled/) on the Sai system, before Single Collateral Dai.
+
+## How was testing for Emergency Shutdown done?
+
+Emergency Shutdown was originally introduced and tested with the Sai prototype deployments. Besides the tests in [the code](https://github.com/makerdao/sai/blob/master/src/sai.t.sol) itself, there were three live runs, all of which happened on deployments of the Sai prototype on the Ethereum Mainnet.
+
+## How will CDP users and Dai holders be notified of Emergency Shutdown?
+
+There will be notices within all official Maker front-ends, as well as announcements on our blog and on all our official communication channels like Twitter, Reddit, Medium, Rocketchat, Telegram, WeChat, and Kakaotalk.
```

Comment:
Please add a single hard return at the end of the file

Reply:
done